plugins/spotify.plugin/install.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-apt install -y software-properties-common
```

Comment:
The command `add-apt-repository` is provided by the package `software-properties-common`, so we have to run this first to be sure the next line runs without error. As other packages also use this, you'll see the installation of `software-properties-common` in several plugins.